SR-15138: Unused Parameter in SILFunction.init()

aleksproger opened the pull request:
**Reason**
 - Fixes SR-15138

**Content**

 - Deleted unused parameter from init()
 - Applied formatting to changed files
 - There also extra loc parameter in fabric function SILFunctionBuilder::createFunction(), but it will affect plenty of files.
 - So firstly I want to merge changes related to the initializer

aleksproger commented on lib/SIL/IR/SILFunction.cpp:
```diff
@@ -138,44 +134,36 @@ static FunctionParseFn parseFunction = nullptr;
-                         IsDynamicallyReplaceable_t isDynamic,
-                         IsExactSelfClass_t isExactSelfClass,
-                         IsDistributed_t isDistributed)
+SILFunction::SILFunction(
```
Deleted loc here, around #line133 and #line159
The rest of the changes is formatting

LucianoPAlmeida replied:
Normally people recommend to only format the lines you changed because makes it easier for the reviewer and keep the commit history for the rest of the file cleaner =] 
You can use either `git-clang-format` tool that is on `${PATH_TO_SWIFT}/llvm-project/clang/tools/clang-format` and you can add it to your path, and run on the git directory after `git add .`
Or `clang-format -lines 1630:1660 lib/Sema/CSFix.h -I` for example to run at lines in specific.

aleksproger commented:
@roop please review, that's my first so hope I do it right:)

Commit:
**Reason**
 - Fixes SR-15138

**Content**

 - Deleted unused parameter from init()
 - Applied formatting to changed files
 - There also extra loc parameter in fabric function SILFunctionBuilder::createFunction(), but it will affect plenty of files.
 - So firstly I want to merge changes related to the initializer
ith '#' will be ignored, and an empty message aborts the commit.

CodaFi commented:
Makes sense to me

Let's see what CI thinks 

@swift-ci smoke test

aleksproger commented:
Failed for mac and linux, but it failed before building, so it does not seem to be related to PR
@CodaFi may you please trigger CI again

LucianoPAlmeida commented:
@swift-ci Please smoke test
